tests/functional/spec/summary-screen.spec.js

```diff
@@ -63,4 +63,28 @@ describe('RSI - summary screen edit test', function() {
     expect(rsiSummaryPage.getChangeInRetailTurnoverSummary()).to.contain('This is to test edit links on summary screen - edited')
 
   })
+  it('Given a 0 is entered in a currency field it should still be 0 on the summary page and the original page it was entered', function(done) {
```

Comment:
For consistency it might be worth continuing with the suggested Given When Then structure examples are here http://martinfowler.com/bliki/GivenWhenThen.html